src/components/FilterBar.test.jsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen } from '@testing-library/react';
+import queryString from 'query-string';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import FilterBar from './FilterBar';
+
+const renderWithTheme = (ui) =>
+  render(<ThemeProvider theme={createTheme()}>{ui}</ThemeProvider>);
+
+const queryOptions = {
```

Comment:
not needed

Reply:
`renderWithTheme` seems to be needed. I ran the code without it and it didnt work. the error that I got mentioned `ThemeProvider`

Reply:
I was refering to the `queryOptions` object (: